docs/pages/guides/webhooks.mdx

```diff
@@ -0,0 +1,147 @@
+---
+meta:
+  title: "Webhooks"
+  parentTitle: "Guides"
+  description: "Learn Liveblocks Webhooks concepts and implementation"
+---
+
+## Webhooks
+
+In this guide, we will explain how to use webhooks within your product, to ensure that you are able to store and process Liveblocks data real time. Understanding how to implement webhooks can make your mulitplayer experience observable, and reduce development time by eliminating the need for polling.
+
+### What is a webhook?
+The main drivers of webhooks implementation are efficiency and observability. Webhooks streamline collecting data between two connected services. 
+
+Like an API, webhooks are a mechanism for a client to recieve data. The main difference between webhooks and APIs is that a webhook functions as an inverted API. In essence, the architecture of a webhook is composed of the following parts: 
+- You create and provide a client endpoint
+- You configure which type of event messages to listen to
+- The server broadcasts a message (event) to the client endpoint 
+- You verify the payload from the event message and send a 2xx back to the server
+ to trigger a task
```

Comment:
Maybe a diagram could make it easier to understand?

Reply:
Ok, I think this may sound better: 
>The lifecycle of a webhook: 
>1. You create and provide a client endpoint, which listens to event messages 
>2. The Liveblocks server broadcasts a message (event) to the client endpoint when an event occurs
>3. You verify the event message payload from the webhook service provider (Svix), and send a 200 response once the webhook has been verified
>4. You use the message payload to trigger a task 

>Steps 2-4 will be repeated for the duration you subscribe to Liveblocks events. 
We use Figma for flow charts so that is going to take me a minute to familiarize.  I'm happy to work through creating that flow chart, but probably will take some time in the morning.